terraform/51-deploymentprocesswithgitdependency/deployment_process.tf

```diff
@@ -87,6 +87,7 @@ resource "octopusdeploy_deployment_process" "test_deployment_process" {
     name                = "Kubectl Action"
     package_requirement = "LetOctopusDecide"
     start_trigger       = "StartAfterPrevious"
+    target_roles        = [ "mycluster" ]
```

Comment:
Target Roles are now validated on save, previously this would blow up during execution (see https://github.com/OctopusDeploy/OctopusDeploy/pull/25046/files)